tests/integration/security/file_mounted_certs/p2p_mtls_test.go

```diff
@@ -46,6 +46,8 @@ const (
 )
 
 func TestClientToServiceTls(t *testing.T) {
+	// https://issues.redhat.com/browse/OSSM-755
+	t.Skip("https://github.com/istio/istio/issues/0")
```

Comment:
We should fix this in upstream at some point. A regex should be ok, instead of hardcoding a url in the linter.

Reply:
they do have a regex, but it only matches `github.com/istio/istio/issues/[0-9]*`. it's a tool that is compiled into their build image. we could clone and patch it, but I figured it's not really worth the effort